emote/callbacks.py

```diff
@@ -502,3 +502,19 @@ def end_cycle(self):
         if reward < self._cutoff:
             raise Exception(f"Reward too low: {reward}")
         raise TrainingShutdownException()
+
+
+class BatchCallback(LoggingMixin, Callback):
+    def __init__(self):
+        super().__init__()
+
+    def begin_batch(self, *args, **kwargs):
+        pass
+
+    @Callback.extend
+    def collect_multiple(self, *args, **kwargs):
```

Comment:
This feels a bit like an implementation detail? Or do we expect collecting to be an important part of the API, as opposed to getting batches?

Reply:
You're right! 
Removed "collect_multiple". The callback now only provides the batch via "get_batch"